src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
You can create new variables for these methods and then use them in a ternary operator
```suggestion
  const filtredPeopleWithCentury = people.filter((human) => human.sex === 'm' && Math.ceil(human.died / 100) === century);
  
  const filtredPeopleWithoutCentury =  people.filter((human) => human.sex === 'm');
  
  const men = century
    ? filtredPeopleWithCentury 
    : filtredPeopleWithoutCentury ;
```